packages/sanes-chrome-extension/src/routes/share-identity/index.tsx

```diff
@@ -31,8 +31,14 @@ const ShareIdentity = ({ location }: RouteComponentProps): JSX.Element => {
 
   return (
     <React.Fragment>
-      {action === 'show' && <ShowRequest onAcceptRequest={onAcceptRequest} showRejectView={showRejectView} />}
-      {action === 'reject' && <RejectRequest onBack={showRequestView} onRejectRequest={onRejectRequest} />}
+      {action === 'show' && (
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
For avoid the constant eslint disable check of `request!` I suggest to refactor code and do:
```
const request = requestContext.firstRequest;
const valid = validRequest(request, location, toast) <-- from checkRequest to validRequest
if (!valid) {
  return;
}
```

Reply:
Yes, that's was one of my first thoughts. But after this check I will be required to assign `request` variable to another non-undefined var.
like: 
```
// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
const requestValid = request!
```
And only after this I can use it in prop without any comment :(